cmd/update/update.go

```diff
@@ -0,0 +1,60 @@
+package update
+
+//This update command updates to the latest kubescape release.
+//Example:-
+//          kubescape update
+
+import (
+	"fmt"
+	"log"
+	"os/exec"
+	"runtime"
+
+	"github.com/kubescape/kubescape/v2/core/cautils"
+	"github.com/spf13/cobra"
+)
+
+func GetUpdateCmd() *cobra.Command {
+	updateCmd := &cobra.Command{
+		Use:   "update",
+		Short: "Update your version",
+		Long:  ``,
+		RunE: func(_ *cobra.Command, args []string) error {
+			vu := cautils.NewIVersionCheckHandlerU()
+			vu.CheckLatestVersionU(cautils.NewVersionCheckRequestU(cautils.BuildNumber, "", "", "version"))
+			//Checking the user's version of kubescape to the latest release
+			if cautils.BuildNumber == cautils.LatestReleaseVersion {
+				//your version == latest version
+				fmt.Println("You are in the latest version")
+			} else {
+				//execute the install.sh if linux, install.ps1 for windows,.....depending on your OS
+				var OSTYPE string = runtime.GOOS
+				if OSTYPE == "linux" {
```

Comment:
How about a switch case?

Reply:
Even I thought to add switch cases but runtime.GOOS don't have the value for mac

Reply:
You can use default.
e.g.
```
switch X{
 case "x":
 default:
}
```